src/NexusMods.App.UI/Pages/CollectionDownload/CollectionDownloadView.axaml

```diff
@@ -46,6 +46,22 @@
                 </MenuItem.Header>
             </MenuItem>
         </MenuFlyout>
+
+        <StackPanel x:Key="TabItemContent" Orientation="Vertical" Spacing="0">
```

Comment:
You shouldn't put a `TreeDataGrid` in a vertical `StackPanel`, because the `StackPanel` would have infinite heigh and that messes with the `ScrollViewer` inside the `TreeDataGrid`.

Either use a different panel or add a note for later